pom.xml

```diff
@@ -1560,6 +1563,7 @@
         <hbase.profile>2.2</hbase.profile>
         <hbase.compat.version>2.2.5</hbase.compat.version>
         <hbase.version>${hbase-2.2.runtime.version}</hbase.version>
+        <hadoop.version>3.1.3</hadoop.version>
```

Comment:
This could be 3.1.2 as latest HBase release 2.2.6 supports it: https://github.com/apache/hbase/blob/rel/2.2.6/pom.xml#L1373